src/dealDashboard/dealVotes/dealVotes.html

```diff
@@ -39,8 +39,8 @@ <h1 class="title heading03Gradient">Vote</h1>
         class="negotiate"
         if.bind="ethereumService.defaultAccountAddress  && !deal.fundingWasInitiated && !deal.isCancelled"
       >
-        <pbutton type="tertiary" click.delegate="goToDiscussions()">
```

Comment:
could we still keep a class?
the style in .scss is a bit too general `pbutton button`